android/res/values/styles.xml

```diff
@@ -37,6 +37,10 @@
     <item name="android:layout_margin">@dimen/routing_selector_wheel_margin</item>
     <item name="android:visibility">invisible</item>
     <item name="android:background">?selectableItemBackgroundBorderless</item>
+    <item name="centerDrawable">@android:color/transparent</item>
+    <item name="wheelProgressColor">?colorAccent</item>
+    <item name="wheelSecondaryColor">?dividerHorizontal</item>
+    <item name="wheelThickness">@dimen/margin_eighth</item>
```

Comment:
На каких-то версиях ОС? 
Как минимум на том, что я тестил - работает.
> On May 27, 2016, at 2:48 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/res/values/styles.xml <https://github.com/mapsme/omim/pull/3340#discussion_r64893465>:
> 
> > @@ -37,6 +37,10 @@
> >      <item name="android:layout_margin">@dimen/routing_selector_wheel_margin</item>
> >      <item name="android:visibility">invisible</item>
> >      <item name="android:background">?selectableItemBackgroundBorderless</item>
> > +    <item name="centerDrawable">@android:color/transparent</item>
> > +    <item name="wheelProgressColor">?colorAccent</item>
> > +    <item name="wheelSecondaryColor">?dividerHorizontal</item>
> > +    <item name="wheelThickness">@dimen/margin_eighth</item>
> Так не будет работать, к сожалению.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/mapsme/omim/pull/3340/files/bbcfdc3177e5b5af0bbc60f08aae133f4792727e#r64893465>, or mute the thread <https://github.com/notifications/unsubscribe/ACKTRNaCqO67wpNSY_AHmJVkx8jnrz22ks5qFtoEgaJpZM4ImvLw>.
>